openscap_report/report_generators/html_templates/rule_detail.html

```diff
@@ -178,7 +178,10 @@
                             <i class="fas fa-fw fa-exclamation-triangle" aria-hidden="true"></i>
                         </div>
                         <div class="pf-c-alert__title">
-                            {{- warning | set_css_for_list -}}
+                            Warning category:&nbsp;{{- warning.category | capitalize -}}
```

Comment:
This looks ugly. Keep only `Warning` as a title. Put the category value to a box or under the heading and don't use the "category" word at all.

Reply:
That sounds great!